doc-surrealdb_versioned_docs/version-2.x/security/authentication.mdx

```diff
@@ -104,23 +104,24 @@ curl -X POST \
 	http://localhost:8000/signin
 ```
 
-## Scope users
+## Record users
 
-Scope is the term we use to describe the mechanism SurrealDB offers to define your own signin and signup logic. This feature contributes to making SurrealDB an all-in-one BaaS (Backend-as-a-Service).
+Record users represent users that are defined as a record in a database instead of through the `DEFINE USER` statement. Since these users exist as regular database records, they can have associated fields containing any information required for authentication and authorization. For this reason, record users allow SurrealDB to offer mechanisms to define your own signin and signup logic as well as custom table and field permissions. This feature contributes to making SurrealDB an all-in-one BaaS (Backend-as-a-Service).
```

Comment:
```suggestion
Record users represent users that are defined as a record in a database instead of through the `DEFINE USER` statement. Since these users exist as regular database records, they can have associated fields containing any information required for authentication and authorization. 

For this reason, SurrealDB offers mechanisms to define your own signin and signup logic as well as custom table and field permissions for record users. This feature contributes to making SurrealDB an all-in-one BaaS (Backend-as-a-Service)..
```

Reply:
Now that I look at this text again, it should be "thanks to this" or "because of this" instead of "for this reason". I will make an additional commit after accepting your suggestion.